UE/QtApplicationEnvironment/GUI/QtUeGui.hpp

```diff
@@ -49,6 +49,7 @@ class QtUeGui : public QObject, public IUeGui
     void showConnecting() override;
     void showNotConnected() override;
     void showNewSms() override;
+    void showNotNewSms() override;
```

Comment:
I don't think we are supposed to change the Qt portion of the project. Can't it be done differently, directly from the UserPort?